doc/train_parameters.md

```diff
@@ -0,0 +1,155 @@
+# Brief design for unifying format of training parameters in SQLFlow
+
+SQLFlow propose an extension to the SQL SELECT statement, which proposes the syntax for training as 
+```sql
+SELECT * FROM kaggle_credit_fraud_training_data
+LIMIT 1000
+TRAIN DNNClassifier       /* a pre-defined TensorFlow estimator, tf.estimator.DNNClassifier */
+WITH layers=[100, 200],   /* a parameter of the Estimator class constructor */
+     train.batch_size = 8 /* a parameter of the Estimator.train method */
+COLUMN *,                 /* all columns as raw features */
+       cross(v1, v9, v28) /* plus a derived (crossed) column */
+LABEL class
+INTO sqlflow_models.my_model_table;      /* saves trained model parameters and features into a table */
+```
+
+The syntax for training must following rules as 
```

Comment:
Currently, we have the following syntax allowing users to provide necessary information for the training.